src/js/editor/editor.js

```diff
@@ -293,7 +306,7 @@ class Editor {
     this.run(postEditor => {
       let nextPosition = postEditor.deleteAtPosition(position, direction, {unit});
       postEditor.setRange(nextPosition);
-    });
+    }, EDITOR_RUN_LAST_ACTION_STATE.DELETE);
```

Comment:
Also, there isn't necessarily anything about the `run` method that requires it to be only one of ["insert","delete"] — it's totally possible to do an `editor.run` that inserts at some places and deletes at others.
Not sure how important this distinction is. The most common use case is certainly for a `run` to focus on either one or the other.